src/compile/legend.ts

```diff
@@ -81,7 +81,7 @@ export function formatMixins(model: Model, channel: Channel) {
   }
 
   const legend = fieldDef.legend;
-  return model.formatMixins(channel, (typeof legend !== 'boolean' && legend.format) || undefined);
+  return model.formatMixins(channel, (typeof legend !== 'boolean' && legend.format));
```

Comment:
formatMixins accept string as second parameter  
 should it be 
`typeof legend !== 'boolean' ? legend.format : undefined`  

although it pretty much gives same result.